numpyro/examples/datasets.py

```diff
@@ -27,7 +27,11 @@
 ])
 
 BASEBALL = dset('baseball', [
-    "https://d2fefpcigoriu7.cloudfront.net/datasets/EfronMorrisBB.txt",
+    'https://d2fefpcigoriu7.cloudfront.net/datasets/EfronMorrisBB.txt',
+])
+
+UCBADMIT = dset('ucbadmit', [
+    'https://raw.githubusercontent.com/rmcelreath/rethinking/master/data/UCBadmit.csv',
```

Comment:
FIXME: should we upload the data to `cloudfront.net`?

Reply:
@fehiepsi - I think that will be better. Let me upload that, and update the link here.

Reply:
You should be able to fetch this from: `https://d2fefpcigoriu7.cloudfront.net/datasets/UCBadmit.csv`. Its such a tiny dataset that it doesn't really matter though. :)

Reply:
Thanks @neerajprad !